SimG4Core/PhysicsLists/python/physicsQGSP_FTFP_BERT_G4104_cfi.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+from SimG4Core.Configuration.SimG4Core_cff import *
+
+g4SimHits.Physics.EmaxBERT   = cms.double(9.9) # in GeV
```

Comment:
This transition is really problematic.

Reply:
This was again a part of Geant4.10.4.p03